string topicName
int64 nImages
---
stereo_msgs/DisparityImage[] images

